app/components/avo/actions_component.rb

```diff
@@ -79,4 +79,45 @@ def action_url(action, path)
         }.compact
       ).to_s
   end
+
+  def icon(action)
+    svg action.icon || "play", class: "h-5 mr-1 inline pointer-events-none"
+  end
+
+  def render_entity(action, next_action)
```

Comment:
How about using `item` instead of `entity`?
It's a bit more friendly and we use that in other places of the codebase.

Reply:
Sounds good. I will replace it.